### ISSUE TITLE
False Positive on mysql-connector-java/8.0.22

### ISSUE THREAD

**Issue by gamma** [dependency-check/DependencyCheck#3041]
False positive on library mysql-connector-java-8.0.22.jar - reported as 

```
cpe:2.3:a:mysql:mysql:8.0.22:*:*:*:*:*:*:*  (Confidence:Highest)
cpe:2.3:a:oracle:connector\/j:8.0.22:*:*:*:*:*:*:*  (Confidence:Highest)
cpe:2.3:a:oracle:jdbc:8.0.22:*:*:*:*:*:*:*  (Confidence:Highest)
cpe:2.3:a:oracle:mysql_connector\/j:8.0.22:*:*:*:*:*:*:*  (Confidence:Highest)
```

probably matched by:

```
cpe:2.3:a:mysql:mysql:*:*:*:*:*:*:*:*
```

```xml
<!-- https://mvnrepository.com/artifact/mysql/mysql-connector-java -->
<dependency>
    <groupId>mysql</groupId>
    <artifactId>mysql-connector-java</artifactId>
    <version>8.0.22</version>
</dependency>
```

**Comment by jeremylong** [comment]
Two of those CPEs would not be FP as Oracle:MySQL Connector is correct. We have also seen that client/connector related CVEs being reported against what might be considered the server (i.e. mysql:mysql). I've added the following suppression rule:

```xml
    <suppress base="true">
        <notes><![CDATA[
        FP per issue #3041
        ]]></notes>
        <packageUrl regex="true">^pkg:maven/mysql/mysql\-connector\-java@.*$</packageUrl>
        <cpe>cpe:/a:oracle:connector%2fj</cpe>
        <cpe>cpe:/a:oracle:jdbc</cpe>
    </suppress>
```
